dev/com.ibm.ws.jsp.2.3_fat/build.gradle

```diff
@@ -9,9 +9,9 @@
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
 dependencies {
-    requiredLibs 'commons-codec:commons-codec:1.11',
-      'commons-codec:commons-codec:1.6',
-      'commons-logging:commons-logging:1.1.3',
+    requiredLibs project(':io.openliberty.org.apache.commons.codec'),
+      project(':io.openliberty.org.apache.commons.codec'),
+      project(':io.openliberty.org.apache.commons.logging'),
```

Comment:
It seems odd that this fat had two versions of commons-codec in the first place, but it certainly doesn't now need the same version declared twice.